imsv-docs-docusaurus/openapi/endpoints/kyc/models/drivers-license.yaml

```diff
@@ -0,0 +1,26 @@
+properties:
+  evidenceType:
+    type: string
+    description: The type of evidence.
+    enum:
+      - DRIVERS_LICENSE
+  documentId:
+    type: string
+    description: The license number.
+    example: DS230475
+  country:
+    type: string
+    description: The country that issued the document.
+    enum:
+      - NZL
+      - AUS
+    example: NZL
+  version:
+    type: string
+    description: The driver's license version
```

Comment:
I should have done it as part of my recent fix: 
- https://github.com/immersve/amethyst/pull/1673
but while you are here, would you please make this description work for AU licenses too, like
"The driver's license version or card number"

Reply:
another ask is to add the freshly added OPTIONAL 'region' field, please 🙏 
```
region:
  type: string
  description: Region of the document (state, county) if applicable. A local abbreviation should be used. For example TX for Texas, US or QLD for Queensland, Australia. 
  example: QLD
```